arangod/IResearch/GeoAnalyzer.cpp

```diff
@@ -23,32 +23,29 @@
 
 #include "IResearch/GeoAnalyzer.h"
 
-#include <string>
-
-#include <s2/s2point_region.h>
-#include <s2/s2latlng.h>
-
-#include <velocypack/Builder.h>
-
 #include "analysis/analyzers.hpp"
 
+#include "Basics/DownCast.h"
+#include "Basics/Exceptions.h"
 #include "Geo/GeoParams.h"
 #include "Geo/GeoJson.h"
+#include "Inspection/VPack.h"
```

Comment:
Is it used?

Reply:
No, it seems it is also not used.
Removed it as well now.